src/SimpleExcelReader.php

```diff
@@ -233,7 +233,7 @@ public function headerRowFormatter(callable $callback)
 
     protected function trim(string $header): string
     {
-        return call_user_func_array('trim', array_filter([$header, $this->trimHeaderCharacters]));
```

Comment:
When `$header` is a nullable or an empty string, `array_filter` will extract it out.

So, when both `$header` and `trimHeaderCharacters` are empty, it will throw the following exception: `ErrorException: trim() expects at least 1 parameter, 0 given`.

It can be even more "dangerous" when the `trimHeaderCharacters` property is not null but a real string like `.txt` or so.